ga4gh/datamodel/variants.py

```diff
@@ -47,6 +48,17 @@ def convertVCFGenotype(vcfGenotype, vcfPhaseset):
     return genotype, phaseset
 
 
+class CompoundVariantId(datamodel.CompoundId):
+    """
+    The compound id for a variant
+    """
+    fields = ['datasetId', 'vsId', 'chrom', 'pos', 'md5']
```

Comment:
We should follow the ga4gh names for this: ``variantSetId``, ``referenceName``, etc.